docs/api/synchronization.rst

```diff
@@ -0,0 +1,160 @@
+.. _api-synchronization:
+
+Synchronization
+###############
+
+This section describes the basic aspects of synchronization using *Kinto*.
+
+.. note::
+
+    If you are looking for a ready-to-use synchronization solution,
+    jump to :ref:`sync-implementations`.
+
+
+The main idea consists in maintaining a local database up-to-date
+with the *Kinto* server. Remote changes are downloaded and applied
+on the local data. Local changes are uploaded using HTTP headers to control
+concurrency and overwrite.
+
+In short:
+
+#. Poll for remote changes using ``?_since=<timestamp>``
+#. Apply changes locally
+#. Post creations
```

Comment:
"Post" has a number of meanings and, contextually, this is a rough place to employ it.
Consider:
> Upload creations.
or
Apply creations *remotely*.